packages/typescript-typings/types/web3/index.d.ts

```diff
@@ -73,7 +73,7 @@ declare module 'web3' {
         }
 
         interface FilterResult {
-            get(callback: () => void): void;
+            get(callback: (err: Error, result: LogEntryEvent[]) => void): void;
```

Comment:
Let's rename `result` to `logs` to more closely match the [docs](https://github.com/ethereum/wiki/wiki/JavaScript-API#example-50).